src/Eloquent/Builder.php

```diff
@@ -49,6 +50,7 @@ class Builder extends EloquentBuilder
         'insertusing',
         'max',
         'min',
+        'autocomplete',
```

Comment:
I noted how [Builder::__call()](https://github.com/illuminate/database/blob/482726b5a5f444321341bae5a1be01212223c1cc/Eloquent/Builder.php#L2042) handles this.

Why does only `autocomplete` get `$passthru`? Is that because it doesn't perform any hydration, so the implementation is identical? That would be in contrast to `search()`, since the method in this class needs to hydrate.

Reply:
- by default, the `Eloquent\Builder` ignores the result of the call to the `Query\Builder` method, and return the `Eloquent\Builder` instance for "fluent API".
- `passthru` is to get the unmodified result of the `Query\Builder` method call.
- by overloading `Query\Builder::search` as `Eloquent\Builder::search`, I can get and hydrate the results.